carbonserver/carbonserver/api/routers/emissions.py

```diff
@@ -77,3 +79,15 @@ def get_emissions_from_run(
     params: Params = Depends(),
 ) -> Page[Emission]:
     return paginate(emission_service.get_emissions_from_run(run_id), params)
+
+
+@router.get("/emissions", tags=EMISSIONS_ROUTER_TAGS, response_model=Page[Emission])
+@inject
+def get_emissions(
+    run_id: str,
+    emission_service: EmissionService = Depends(
+        Provide[ServerContainer.emission_service]
+    ),
+    params: Params = Depends(),
+) -> Page[Emission]:
+    return paginate(emission_service.get_emissions_from_run(run_id), params)
```

Comment:
what if no run_id is provided in the query? How to handle that?

Reply:
run_id is mandatory so you can access emissions by either:
- /runs/{runs[0]['id']}/emissions
- /emissions?run_id=...

If you don't specify run_id in the second variant, you'll get a fastapi validation error